nsv13/code/game/gamemodes/overmap/shakedown.dm

```diff
@@ -17,23 +17,9 @@
 	max_players = 10
 	random_objective_amount = 3
 
-/datum/overmap_gamemode/New()
+/datum/overmap_gamemode/shakedown/New()
```

Comment:
It's probably good to define overmap_gamemode/New() somewhere else with the old values so spawning a new generic  gamemode won't instantly end due to lack of objectives.